src/NewRelic.Telemetry/Extensions/DateTimeExtensions.cs

```diff
@@ -9,7 +9,7 @@ namespace NewRelic.Telemetry.Extensions
     /// Provides DateTime conversions to Unix Timestamps.
     /// This supports backwards compatibility with .NET v4.5.
     /// </summary>
-    internal static class DateTimeExtensions
+    public static class DateTimeExtensions
```

Comment:
Since we've removed builder methods, this should be made public to enable users to perform these conversions easily.

Reply:
Really, these extensions are only helpful for `net452`, I'd be inclined to keep it internal for now and _maybe_ ship it in a `net452` compilation only. Anyone running `net46+` will have this functionality.